docs/guides/cli.md

```diff
@@ -61,6 +62,67 @@ Connection: keep-alive
 
 The body, headers, etc. for this response will be taken from the API description document.
 
+#### Request Specific Examples
+
+You can request a specific example from your document by using the `__example` query string parameter.
+
+```bash
+curl -v http://127.0.0.1:4010/pets/123?__example=exampleKey
+```
+
+#### Dynamic Response
+
+You can override the `--dynamic|-d` CLI param (which decides whether the generated example is static or dynamic) through the `__dynamic` query string parameter.
+
+```bash
+curl -v http://127.0.0.1:4010/pets/123?__dynamic=false
+```
+
+## Proxy
+
+This command creates an HTTP server that will proxy all the requests to the specified upstream server. Prism will analyze the request coming in and the response coming back from the upstream server and report the discrepancies with what's declared in the provided OpenAPI document.
+
+**Note:** Currently the proxy command _expects_ a JSON Payload for both the request and the response. If the response's `content-type` header does not indicate a JSON-ish payload (`application/json`, `application/*+json`), then the payload will be just returned as text and send back to the original client.
+
+```
+prism proxy examples/petstore.oas2.yaml https://petstore.swagger.io/v2
+
+[CLI] …  awaiting  Starting Prism…
+[HTTP SERVER] ℹ  info      Server listening at http://127.0.0.1:4010
+[CLI] ●  note      GET        http://127.0.0.1:4010/pets
+[CLI] ●  note      POST       http://127.0.0.1:4010/pets
+[CLI] ●  note      GET        http://127.0.0.1:4010/pets/10
+```
+
+The output violations will be reported on the standard output and as a response header (`sl-violations`)
+
+```bash
+prism proxy examples/petstore.oas2.yaml https://petstore.swagger.io/v2
+… …
+
+curl -v -s http://localhost:4010/pet/10 > /dev/null
+
+< sl-violations: [{"location":["request"],"severity":"Error","code":401,"message":"Invalid security scheme used"}]
+```
+
+You can see there's a `sl-violations` header which is a JSON object with all the violations found in the response.
+
+In case you want to return an error HTTP response in case there's an `Error` in the request or the response, you can use the `--errors` flag.
+
+```bash
+prism proxy examples/petstore.oas2.yaml https://petstore.swagger.io/v2 --errors
+… …
+
+curl -v -X POST http://localhost:4010/pet/
+
+< HTTP/1.1 422 Unprocessable Entity
+{"type":"https://stoplight.io/prism/errors#UNPROCESSABLE_ENTITY","title":"Invalid request body payload","status":422,"detail":"Your request/response is not valid and the --errors flag is set, so Prism is generating this error for you.","validation":[{"location":["request"],"severity":"Error","code":401,"message":"Invalid security scheme used"}]}
```

Comment:
@philsturgeon I'm starting to think that 422 is not a good status code, since this error can either be because of an invalid request but also because of an invalid response. Any idea?

Reply:
Ahh yeah, so if we have a violation on a response it can be a 500, as the client is not at fault.

Reply:
I think my comment got lost somewhere but agree, response violations = 500

Reply:
500 it is.